Example/OriginateUI/AppDelegate.m

```diff
@@ -7,21 +7,33 @@
 //
 
 #import "AppDelegate.h"
-#import "MyViewController.h"
+#import "HomeViewController.h"
 
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
-
-    MyViewController *themableViewController = [[MyViewController alloc] init];
-    UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:themableViewController];
-    
+    HomeViewController *homeViewController = [[HomeViewController alloc] init];
```

Comment:
Let's try to keep the didFinishLaunchingWithOptions: method slim. So we should lazy load the navigationController in a property method and there initialize it.